models/Profile.php

```diff
@@ -47,6 +56,7 @@ public function rules()
     {
         return [
             'bioString' => ['bio', 'string'],
+            'profileString' => ['timezone', 'string', 'max' => 255],
```

Comment:
I think we need more complex rule here: we should check if the timezone really exist.